### ISSUE TITLE
Provide anaconda support

### ISSUE THREAD

**Issue by BenjaminRodenberg** [precice/python-bindings#84]
[anaconda](https://anaconda.org/) is a big thing in the python community, but currently not supported by the python bindings. I personally lack knowledge about anaconda, but would be very thankful, if somebody from the community has feedback on this issue.

My impression is that often users are already using anaconda when they "discover" preCICE and would like to continue using anaconda.

Possible problems I currently see are compatibility issues with preCICE and/or spack. For FEniCS it might be nice to also support anaconda, since anaconda is one of the [recommended ways of installing FEniCS](https://fenicsproject.org/download/).

**Comment by ajaust** [comment]
In what sense do the bindings not work with anaconda? Could you elaborate on that? Is it impossible to install the bindings from hand or by using `pip`? What is the error appearing?

Mixing package managers is often a problem and also the anaconda developers warn of doing that. However, there is a post from 2018 (https://www.anaconda.com/blog/using-pip-in-a-conda-environment) elaborating on how to mix `pip` and anaconda.

**Comment by ajaust** [comment]
What is the current status of the anaconda support?

There is a (ana)conda package called `pyprecice` available on [conda-forge](https://anaconda.org/conda-forge/pyprecice). However, when a colleague tried to use it on Windows via the WSL it failed. Importing the module in a Python shell via `import precice` led to a MPI initialization error.

I cannot test it myself at the moment since I do not have a Windows installation available.

**Comment by BenjaminRodenberg** [comment]
Hi @ajaust,

thanks for bringing this to our attention! We already added a pointer to the community-driven anaconda packages on our website under the [installation instructions for the python bindings](https://precice.org/installation-bindings-python.html#alternatives--community-packages). I tried out the anaconda packages for `v2.3.0.1` recently and everything worked nicely.

From my perspective this closes this issue. Feel free to comment and reopen, if you see something that is still missing.
